tests/components/sonos/test_media_player.py

```diff
@@ -16,6 +16,7 @@
     DeviceRegistry,
 )
 
+from homeassistant.components.sonos.media_player import SERVICE_PARTYMODE
```

Comment:
Imports need to be sorted alphabetically, running black will fix.  So this needs to be moved in front of

```
from homeassistant.const import STATE_IDLE
```